src/ui/state/timeline.ts

```diff
@@ -33,8 +33,8 @@ export interface TimelineState {
 
 export interface HoveredPoint {
   target: "timeline" | "console" | "widget" | "transcript";
-  point: string;
-  time: number;
+  point?: string;
+  time?: number;
```

Comment:
i think all hovered points should have these properties unless there's a really good reason

Reply:
Well, if you hover over a breakpoint panel, you have a location but no singular time.

Reply:
got it... i don't think we should consider that a point.

Reply:
Yeah, the name `HoveredPoint` seems wrong here, but we still want the functionality when hovering over a breakpoint panel, right?

Reply:
haha we're debating that actually. But either way we want it when hovering on the gutter. 

I propose introducing a new hoveredLocation type and state. It's nice to keep this strict.

Reply:
> I propose introducing a new hoveredLocation type and state.

I don't think that's a good idea:
* location and point/time in `HoveredPoint` come from the same thing in the UI (the one you're hovering over)
* most parts of the code that use `HoveredPoint` read or set both location and point/time at the same time
* the `target` property of `HoveredPoint` would have to be duplicated - or stored separately, in which case we would have three separate entities in redux (coming from one thing in the UI) instead of one entity with 4 properties